core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java

```diff
@@ -11971,4 +11971,50 @@ RelNode createLogicalValueRel(RexNode col1, RexNode col2) {
     assertThat(toSql(root, DatabaseProduct.ORACLE.getDialect()), isLinux(expectedOracle));
   }
 
+
+  @Test public void testForGetBitFunction() {
+    final RelBuilder builder = relBuilder();
+    final RexNode getBit = builder.call(SqlLibraryOperators.GETBIT,
```

Comment:
can we please rename this to getBitRexNode

Reply:
Done